Expensify/App
# Fix Issue #1039 : extending KeyboardShortcut to Support Special Keys

<If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

### Details
Added Escape key listener to close all modals using existing KeyboardShortcut library. KeyboardShortcut library was also modified to accept single keys by passing `special` as modifier when subscribing for event.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/1039

### Tests

Close Image upload preview:

1. Go to chat screen of Expensify.cash
2. Click on image upload button and select image for preview
3. Press Escape button and modal should close

Close Attachment preview modal:
1. Press on any image in chat
2. Wait for preview modal
3. Press Escape and modal should close


### Tested On


### Screenshots

https://user-images.githubusercontent.com/26269088/103719406-e0dfa380-4fc0-11eb-9bcc-f145b7ed4377.mp4
